contracts/src/v0.7/Operator.sol

```diff
@@ -39,13 +39,14 @@ contract Operator is
   // We initialize fields to 1 instead of 0 so that the first invocation
```

Comment:
Contracts changelog entry needed for this.

Reply:
Can you add a reference in the changelog to the fact that Operator and ChainlinkClient in v0.7 has been updated with these changes, but not v0.6 or v0.8 yet?

Reply:
Updated.